src/esp/sensor/Sensor.h

```diff
@@ -78,6 +78,10 @@ class Sensor : public Magnum::SceneGraph::AbstractFeature3D {
 
   SensorSpec::ptr specification() const { return spec_; }
 
+  inline Magnum::Vector2i framebufferSize() const {
+    return {spec_->resolution[1], spec_->resolution[0]};
```

Comment:
I would put a **big** comment here pointing out the order. Because it took me an awful lot of time to realize that :)